src/pages/new/donate/index.js

```diff
@@ -1,5 +1,7 @@
 import React from "react";
 
+import ArticleSpotlightCard from "../../../components/grid-aware/ArticleSpotlightCard";
+import articleSpotlightImage from "../../../components/grid-aware/ArticleSpotlightCard/mission-hotel.jpeg";
```

Comment:
Instead of putting the image in the component directory, could you put it in the directory containing the page itself? The images are not part of the generic components, and they are specific to the instances of the component on the individual pages.